README.md

```diff
@@ -3,4 +3,17 @@
 This is an ammended version of the GoCardless Pro Ruby example to work with the new API.
 
 The app passes details to the GoCardless api to retrieve a redirect flow, and set up customers with mandates
-to pay the Fred Roche Foundation for their annual friends programme.
\ No newline at end of file
+to pay the Fred Roche Foundation for their annual friends programme.
+
+## Running the app locally
+
+```
+git clone git@github.com:matterttack/frf-payment-site.git
+cd frf-payment-site
+bundle install
```

Comment:
You'll need to be using bundler to follow these install instructions, but you're probably already set up with that? `gem install bundler` if you're not.